isis/tests/Fixtures.cpp

```diff
@@ -1693,4 +1693,14 @@ namespace Isis {
       testCube = new Cube(testPath, "rw");
     }
   }
+
+void NearMsiCameraCube::Init() {
+    TempTestingFiles::SetUp();
+
+    QString msicube = "m0155881376f3_2p_cif_dbl.cub";
```

Comment:
This doesn't appear to be in the PR. We also usually don't store full cubes for tests. We generate an ISD and store that + the cube label. Then, we generate the cube at run time.

Reply:
How do I generate an ISD for a NEAR/MSI image?

For now, I have unignored this cube in the repo.

Reply:
It may require some extra steps. I'll spend some time tomorrow seeing if I can do it with the cube you pushed. Can you make sure you have the "allow edits and access to secrets from maintainers" box checked and I can push it up.
![Screen Shot 2022-04-05 at 4 04 12 PM](https://user-images.githubusercontent.com/22942817/161865119-afb7f404-47da-4fec-a470-e9a0dcebf358.png)

Reply:
It is checked.

Thanks...